src/pages/settings/Wallet/AddDebitCardPage.js

```diff
@@ -194,6 +194,7 @@ function DebitCardPage(props) {
                         </Text>
                     )}
                     style={[styles.mt4]}
+                    defaultValue={false}
```

Comment:
This added to solve the below console error, otherwise due to undefined it will set empty string for boolean.

<img width="1439" alt="Screenshot 2023-10-25 at 21 37 28" src="https://github.com/Expensify/App/assets/14358475/b26a1851-fe99-4037-ab42-f7ef612fe0bf">